server.go

```diff
@@ -342,21 +343,43 @@ func (sp *serverPeer) relayTxDisabled() bool {
 
 // pushAddrMsg sends an addr message to the connected peer using the provided
 // addresses.
-func (sp *serverPeer) pushAddrMsg(addresses []*wire.NetAddress) {
-	// Filter addresses already known to the peer.
+func (sp *serverPeer) pushAddrMsg(addresses []*wire.NetAddressV2) {
```

Comment:
I think we should update the `godoc` and/or the method name toe reflect that this pushes the _legacy_ message to the peer.

Reply:
Updated godoc